CefSharp.Wpf/ChromiumWebBrowser.cs

```diff
@@ -506,6 +506,21 @@ private static void OnIsBrowserInitializedChanged(DependencyObject d, Dependency
 
         protected virtual void OnIsBrowserInitializedChanged(bool oldValue, bool newValue)
         {
+            managedCefBrowserAdapter.GetZoomLevelAsync()
+                .ContinueWith(new Action<Task<double>>((Task<double> previous) =>
+                    {
+                        if (previous.IsCompleted)
+                        {
+                            UiThreadRunAsync(() =>
+                            {
+                                SetValue(ZoomLevelProperty, previous.Result);
```

Comment:
I did have a quick look to see what command line options there were, very few referencing `zoom`.

`--force-device-scale-factor` is the only other that I'm aware of and it doesn't appear to change the zoom level.

http://peter.sh/experiments/chromium-command-line-switches/